src/software/ai/hl/stp/tactic/crease_defender/crease_defender_fsm.cpp

```diff
@@ -35,7 +35,7 @@ void CreaseDefenderFSM::blockThreat(
     auto block_threat_point = findBlockThreatPoint(
         event.common.world.field(), event.control_params.enemy_threat_origin,
         event.control_params.crease_defender_alignment,
-        robot_navigation_obstacle_config.robot_obstacle_inflation_factor());
+        robot_navigation_obstacle_config.robot_obstacle_inflation_factor() + 0.5);
```

Comment:
why do we need an additional inflation factor here? why isn't `robot_obstacle_inflation_factor` good enough?

Reply:
after talking to nima: it's so that the crease defenders are more stable on the enemy defense line